src/utilities.h

```diff
@@ -431,23 +431,21 @@ bool isStandardPrefixName(const std::string &name);
 size_t getVariableIndexInComponent(const ComponentPtr &component, const VariablePtr &variable);
 
 /**
- * @brief Test to determine if @p variable1 and @p variable2 are the same or
- * (directly or indirectly) equivalent.
+ * @brief Test to determine if @p variable1 and @p variable2 are equivalent.
  *
- * Test to see if @p variable1 is the same or (directly or indirectly)
- * equivalent to @p variable2. Returns @c true if @p variable1 is the same or
- * (directly or indirectly) equivalent to @p variable2 and @c false otherwise.
+ * Test to see if @p variable1 is the same or equivalent to @p variable2.
```

Comment:
... the same AS or equivalent to ... (also below)

Reply:
Fixed in commit f1f8262c4.